infrahub_sdk/client.py

```diff
@@ -1282,6 +1282,62 @@ async def get_diff_summary(
 
         return node_diffs
 
+    async def get_diff_tree(
```

Comment:
Do you see a value from an SDK point of view to have this as a method directly under the client? I don't have a strong objection to it but the client file is already massive as it is and at some point I'd like to break it apart into smaller components. Don't know if this command would remain in the same place at that point. Not really ready for that refactoring just now. But if we don't see a clear use case to have this within the SDK itself I'd suggest that we start out by just having it in the CTL.